gradle/missing-javadoc.gradle

```diff
@@ -107,6 +107,7 @@ configure([
   project(":modules:aggs-matrix-stats"),
   project(":modules:analysis-common"),
   project(":modules:geo"),
+  project(":modules:identity-shiro"),
```

Comment:
Can we remove this exclusion?

Reply:
Yeah no problem.

Reply:
This will have to stay in. I added javadocs everywhere but the compiler still complains about missing javadocs even when there are simply implied default constructors. This does not make much sense but it does not look like we are currently able to easily configure these settings. I may have missed it etc. but from what I saw, we would need to swap the settings for everything to change the strict-ness level.